.template/addons/github/.github/wiki/_Sidebar.md

```diff
@@ -2,6 +2,7 @@
 
 - [[Home]]
 - [[Getting Started]]
+- [[Contribution Guide]]
 
```

Comment:
```suggestion
- [[Testing]]

```

Then we can add testing too 😇

Reply:
Testing is linked under `Operations` section. Should it also be added here?

Reply:
Yes, as this is a root page of the wiki ;-)